src/redux/reducers/sessionReducer.js

```diff
@@ -10,6 +10,7 @@ export default function sessionReducer(state = initialState, action) {
         loading: true,
       };
     case sessionActions.LOGIN:
+      console.trace();
```

Comment:
You forgot to delete this `console.trace()`

Reply:
Done